node/src/components/consensus/config.rs

```diff
@@ -11,3 +18,32 @@ pub struct Config {
     /// Path to secret key file.
     pub secret_key_path: External<SecretKey>,
 }
+
+/// Consensus protocol configuration.
+#[derive(DataSize, Debug)]
+pub(crate) struct ProtocolConfig {
+    pub(crate) highway_config: HighwayConfig,
+    /// Number of eras before an auction actually defines the set of validators.
+    /// If you bond with a sufficient bid in era N, you will be a validator in era N +
+    /// auction_delay + 1
+    pub(crate) auction_delay: u64,
+    pub(crate) unbonding_delay: EraId,
```

Comment:
That should either be a `u64`, too, or we'd need an `EraIdDiff` type.

Reply:
I used the types that were in the original `Chainspec`. Should they be changed there as well?

Reply:
I think so. I find it misleading to use `EraId` for a _number_ of eras, i.e. a _difference_ of IDs.

Reply:
Actually, the `EraId` here is just a type alias that is really a `u64`.

Reply:
Done in [`3703796` (#665)](https://github.com/CasperLabs/casper-node/pull/665/commits/3703796ffe51d59e32e7d93af6573e4a1a4a0eea).

Reply:
Thanks! Yes, even if it's just a type alias I'd use `u64` for era ID differences then. :+1: